wolfcrypt/test/test.c

```diff
@@ -16702,6 +16702,9 @@ int pkcs7encrypted_test(void)
         pkcs7.unprotectedAttribs   = testVectors[i].attribs;
         pkcs7.unprotectedAttribsSz = testVectors[i].attribsSz;
         pkcs7.heap                 = HEAP_HINT;
+#ifdef ASN_BER_TO_DER
```

Comment:
Recommend doing an `wc_PKCS7_Init` above and remove this. Our test examples should do the init, so the PKCS7 struct is memset to 0.

Reply:
Fixed